deeprank_gnn/NeuralNet.py

```diff
@@ -293,39 +342,41 @@ def train(self, nepoch=1, validate=False, save_model='last',
 
-
                 # Validate the model
                 if validate:
 
-                    t0 = time()
+                    # t0 = time()
                     loss_ = self.eval(self.valid_loader, epoch, "validation")
-                    t = time() - t0
+                    # t = time() - t0
 
                     valid_losses.append(loss_)
 
-                    # save the best model (i.e. lowest loss value on validation data)
+                    # save the best model (i.e. lowest loss value on validation
+                    # data)
                     if save_model == 'best':
 
-                        if min(valid_losses) == _val_loss:
-                            self.save_model(filename='t{}_y{}_b{}_e{}_lr{}_{}.pth.tar'.format(
-                                self.task, self.target, str(self.batch_size), str(nepoch), str(self.lr), str(epoch)))
+                        if min(valid_losses) == _val_loss: # noqa
+                            self.save_model(
+                                filename=f't{self.task}_y{self.target}_b{str(self.batch_size)}_e{str(nepoch)}_lr{str(self.lr)}_{str(epoch)}.pth.tar')
                 else:
-                    # if no validation set, saves the best performing model on the traing set
+                    # if no validation set, saves the best performing model on
+                    # the traing set
                     if save_model == 'best':
-                        if min(train_losses) == _loss:
-                            _log.warning("""The training set is used both for learning and model selection.
+                        if min(train_losses) == _loss: # noqa
+                            _log.warning(
+                                """The training set is used both for learning and model selection.
                                             This may lead to training set data overfitting.
                                             We advice you to use an external validation set.""")
 
-                            self.save_model(filename='t{}_y{}_b{}_e{}_lr{}_{}.pth.tar'.format(
-                                self.task, self.target, str(self.batch_size), str(nepoch), str(self.lr), str(epoch)))
+                            self.save_model(
+                                filename=f't{self.task}_y{self.target}_b{str(self.batch_size)}_e{str(nepoch)}_lr{str(self.lr)}_{str(epoch)}.pth.tar')
 
             # Save the last model
             if save_model == 'last':
-                self.save_model(filename='t{}_y{}_b{}_e{}_lr{}.pth.tar'.format(
-                    self.task, self.target, str(self.batch_size), str(nepoch), str(self.lr)))
+                self.save_model(
+                    filename=f't{self.task}_y{self.target}_b{str(self.batch_size)}_e{str(nepoch)}_lr{str(self.lr)}.pth.tar')
 
-    def test(self, database_test=None, threshold=4, hdf5='test_data.hdf5'):
+    def test(self, database_test=None, threshold=4, hdf5='test_data.hdf5'): # noqa
```

Comment:
`threshold` and `hdf5` arguments are not used. Should I remove them?

Reply:
yes please.